docs/gatsby-config.js

```diff
@@ -58,6 +58,10 @@ module.exports = {
             'advanced/http-engine',
             'advanced/no-runtime',
           ],
+          Testing: [
```

Comment:
This seems like a sensible-enough category but happy to revert as well if it doesn't feel right

Reply:
Yesss 🤩